### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by bravosierrasierra** [openstack-exporter/openstack-exporter#63]
Hello! Im just install golang, set GOPATH and call "make". I see many packages, downloaded as dependencies to $GOPATH/mod/

Build failed with message:

```
build github.com/prometheus/client_golang/prometheus/promhttp: cannot load github.com/prometheus/client_golang/prometheus/promhttp: open /home/stalker/temp/openstack-exporter/vendor/github.com/prometheus/client_golang/prometheus/promhttp: no such file or directory
build github.com/prometheus/common/log: cannot load github.com/prometheus/common/log: open /home/stalker/temp/openstack-exporter/vendor/github.com/prometheus/common/log: no such file or directory
build github.com/prometheus/common/version: cannot load github.com/prometheus/common/version: open /home/stalker/temp/openstack-exporter/vendor/github.com/prometheus/common/version: no such file or directory
!! command failed: build -o /home/stalker/temp/openstack-exporter/openstack-exporter -ldflags -X main.Version=0.7.0 -extldflags '-static' -mod=vendor github.com/openstack-exporter/openstack-exporter: exit status 1
```

**Comment by bravosierrasierra** [comment]
I'm try find solution to disable "INFO" level logging: at that level i get in syslog too many insufficient info for me. 

PS: Sorry for my bad english

**Comment by niedbalski** [comment]
Fixed on #65 

Thanks for reporting @bravosierrasierra !
